examples/website-demos/social-messaging/src/components/ChatContainer/ChatContainer.tsx

```diff
@@ -64,10 +64,10 @@ export const ChatContainer: React.FC = () => {
   const [chatClient, setChatClient] = useState<StreamChat>();
 
   const {
-    chatsUnreadCount,
+    // chatsUnreadCount,
```

Comment:
do we need to keep these commented out values in the file?

Reply:
same as above comment